### DISCUSSION TITLE
Common reproduction issue: This commit does not belong to any branch on this repository...

### DISCUSSION THREAD

**Discussion by lintool** [castorini/anserini#2428] [General]
When I navigate to the URL in your reproduction, I should _not_ get this error.

<img width="1575" alt="Screen Shot 2024-03-28 at 7 10 52 AM" src="https://github.com/castorini/anserini/assets/313837/045c80bb-9792-4034-a613-1477f0d7fcfc">

Your URL should refer to a commit point on the master branch of Anserini/Pyserini.

**Comment by k22mitta** [comment]
Fixed, should work now.
